data/categories.txt

```diff
@@ -2599,100 +2599,100 @@ el:Ενοικίαση αυτοκινήτων|αυτοκίνητο|ενοικιά
 sk:3Autopožičovňa|požičovňa áut
 sw:Kodi ya gari|gari|kodi
 
-amenity-cinema
+amenity-cinema|@entertainment
```

Comment:
Again, as you reference '@entertainment' here, please, get rid of duplicates. Here, and in other cases.